projects/lib-project/src/lib/dock/view-module/layout/layout.component.ts

```diff
@@ -85,6 +103,7 @@ export class LayoutComponent {
 
   onButtonClick(buttonTitle: string) {
     switch (buttonTitle) {
+      case "SAVE_CHANGES":
```

Comment:
why like this ?

Reply:
According to story , we have to show save changes and save as draft 2 buttons 
for this both we are calling same api so kept this condition